src/fundus/scraping/html.py

```diff
@@ -217,7 +216,7 @@ async def fetch(self) -> AsyncIterator[HTML]:
                     response.raise_for_status()
 
                 except (HTTPError, ClientError, HttpProcessingError, UnicodeError) as error:
-                    basic_logger.info(f"Skipped requested URL '{url}' because of '{error}'")
+                    basic_logger.info(f"Skipped requested URL {repr(url)} because of {repr(error)}")
```

Comment:
`'{error}'` is not equal to `{repr(error)}` and should stay untouched.

Reply:
Has been removed.